app/views/registrations/_form.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= form_with model: @user, url: registration_path, class: "form-container flex flex-col gap-6 p-6 bg-white shadow-lg rounded-lg max-w-md mx-auto" do |f| %>
+  <div class="form-header">
+    <h2 class="text-2xl font-semibold text-gray-800 text-center"><%= t('.title') %></h2>
+  </div>
+
+  <div class="form-inputs">
+    <div class="input-group flex flex-col gap-2">
+      <%= f.label :email_address, class: "text-sm font-medium text-gray-700" %>
+      <%= f.email_field :email_address, class: "input-field border border-gray-300 rounded-lg p-3 focus:outline-none focus:ring-2 focus:ring-teal-500 transition" %>
+      
+      <% if @user.errors[:email_address].any? %>
+        <p class="text-red-500 text-sm mt-1">Email <%= @user.errors[:email_address].first %></p>
+      <% end %>
+    </div>
+
+
+    <div class="input-group flex flex-col gap-2">
+      <%= f.label :password, class: "text-sm font-medium text-gray-700" %>
+      <%= f.password_field :password, class: "input-field border border-gray-300 rounded-lg p-3 focus:outline-none focus:ring-2 focus:ring-teal-500 transition" %>
+      
+      <% if @user.errors[:password].any? %>
+        <p class="text-red-500 text-sm mt-1">Password <%= @user.errors[:password].first %></p>
```

Comment:
.first demek doğru değil burada çünkü birden fazla hata gelebilir. Bu mesajlara erişmek için de şunu kullanabiliriz;
`@user.errors.full_messages_for(:password)` daha fazla bilgi için; https://api.rubyonrails.org/classes/ActiveModel/Errors.html